packages/fxa-auth-server/scripts/create-mock-iap-subscriptions/mock-iap-subscription-manager.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { CollectionReference, Firestore } from '@google-cloud/firestore';
+import { AppConfig, AuthFirestore } from 'fxa-auth-server/lib/types';
```

Comment:
I don't think this will cause any issues in this case, but just noticed the import style that gave us some issues last time.

Reply:
This won't be run in production, only on dev.